src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/ObservationMapper.java

```diff
@@ -58,7 +63,8 @@
 
     @RequiredArgsConstructor
     private class InternalMapper {
-        private static final String CODING_SYSTEM = "http://loinc.org";
+        private static final String SNOMED_CODING_SYSTEM = "http://snomed.info/sct";
```

Comment:
SNOMED and Read2 system names are commonly used across various mappers. Worth extracting into one place

Reply:
👍  Created a global constants file we can add values like this to